src/status_im/contexts/wallet/account/view.cljs

```diff
@@ -28,7 +28,6 @@
       (let [{:keys [name color formatted-balance
                     watch-only?]} (rf/sub [:wallet/current-viewing-account])
             customization-color   (rf/sub [:profile/customization-color])]
-        (rn/use-unmount #(rf/dispatch [:wallet/close-account-page]))
```

Comment:
Removed, this change makes really difficult to develop, each time the hot reload is triggered in this namespace, the app navigates to the root of the app.

CC: @OmarBasem